aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1158,12 +1164,17 @@ Tensor& argmax_out(const Tensor& self, c10::optional<int64_t> dim, bool keepdim,
     }
     in = self;
   } else {
+    if (self.numel() == 0) {
+      TORCH_CHECK_INDEX(false, "argmax_out(): Expected reduction dim for self.numel() == 0.");
+    }
```

Comment:
nit: `TORCH_CHECK_INDEX(self.numel() == 0, "argmax_out(): Expected reduction dim for self.numel() == 0.");`

Reply:
Done.